### ISSUE TITLE
What is the difference between train and finetune?

### ISSUE THREAD

**Issue by wgb-256** [CVN-OPIS/MotorNN#17]
Hello,

I read your interesting paper recently and I want to reproduce your project.
As far as I see [summoner.py](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/AAAI2020_release/motor_dynamics/summoner.py), in order to reproduce, I should make 3 steps; train, finetune and test.
Then [train.py](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/AAAI2020_release/motor_dynamics/train.py) and [finetune.py](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/AAAI2020_release/motor_dynamics/finetune.py) have some different lines especially in epoch loops.
What is the difference between train and fintune in your project?

**Comment by sagarverma** [comment]
Hi, 

Thanks for taking interest in our work. The only difference is line 15 and 15 in finetune.py https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/1e1aef9621b4ea71c2ded03ffd7a4a7f53b0326a/motor_dynamics/finetune.py#L15. These lines make sure that we use weights from training done on simulated train set and we load real data trainin set for finetuning. There are other lines that are different but can be made same as train.py.

**Comment by wgb-256** [comment]
@sagarverma 
Thank you for replying!
Got it. Then from line [29](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/1e1aef9621b4ea71c2ded03ffd7a4a7f53b0326a/motor_dynamics/finetune.py#L29) to [39](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/1e1aef9621b4ea71c2ded03ffd7a4a7f53b0326a/motor_dynamics/finetune.py#L39) in [finetune.py](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/AAAI2020_release/motor_dynamics/finetune.py) and from line [34](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/1e1aef9621b4ea71c2ded03ffd7a4a7f53b0326a/motor_dynamics/train.py#L34) to [53](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/1e1aef9621b4ea71c2ded03ffd7a4a7f53b0326a/motor_dynamics/train.py#L53) in [train.py](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/1e1aef9621b4ea71c2ded03ffd7a4a7f53b0326a/motor_dynamics/train.py) should be the same?
(And validation step as well?)
I got an error when I tried to run finetune.

**Comment by sagarverma** [comment]
Yes these lines should be same. Can you please post the error stack trace here?

**Comment by wgb-256** [comment]
Sorry. I forgot saving the errors. It looked reference errors like `get_finetune_loaders was not imported`.
Quick question, is the contents of [finetune.py](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/AAAI2020_release/motor_dynamics/finetune.py) right? I just thought it should be that of [test.py](https://github.com/sagarverma/MotorDynamics_EncoderDecoder/blob/AAAI2020_release/motor_dynamics/test.py). 
Can you doublecheck them?

**Comment by wgb-256** [comment]
@sagarverma 
Any updates?

**Comment by wen-coder** [comment]
Hello

I recently read your interesting paper and I would like to replicate your project.
But I can't download your dataset, is there something wrong with your dataset, can you post a link to the latest dataset
